apps/web/components/imports/schema/ColumnsTable.tsx

```diff
@@ -69,40 +80,107 @@ export function ColumnsTable({ templateId }: ColumnsTableProps) {
+                                    onChange([...(Array.isArray(value) ? value : []), newItem]);
+
+                                    return newItem;
+                                  }}
+                                  onChange={onChange}
+                                />
+                              )}
+                            />
+                          ) : null}
+                          {typeValue === 'Date' ? (
+                            <Controller
+                              name="dateFormats"
+                              control={control}
+                              render={({ field: { value, onChange } }) => (
+                                <MultiSelect
+                                  creatable
+                                  clearable
+                                  searchable
+                                  value={value}
+                                  placeholder="Valid Date Formats, i.e. DD/MM/YYYY, DD/MM/YY"
+                                  data={[
+                                    'DD/MM/YYYY',
+                                    'DD/MM/YY',
+                                    'MM/DD/YYYY',
+                                    'MM/DD/YY',
+                                    ...(Array.isArray(value) ? value : []),
+                                  ]}
+                                  getCreateLabel={(query) => `Add "${query}"`}
+                                  onCreate={(newItem) => {
+                                    onChange([...(Array.isArray(value) ? value : []), newItem]);
+
+                                    return newItem;
+                                  }}
+                                  onChange={onChange}
+                                />
+                              )}
+                            />
+                          ) : null}
+                        </>
+                      );
+                    }}
                   />
-                </td>
-                <td>
-                  <Checkbox title="Is Required?" {...register('isRequired')} />
-                </td>
-                <td>
-                  <Checkbox title="Is Unique?" {...register('isUnique')} />
-                </td>
-                <td>
-                  <Flex gap="xs" justify="flex-end">
-                    <ActionIcon color="blue" type="submit" loading={isColumnCreateLoading}>
-                      <CheckIcon color={colors.blue} />
-                    </ActionIcon>
-                    <ActionIcon color="red" onClick={() => setShowAddRow(false)}>
-                      <CloseIcon />
-                    </ActionIcon>
-                  </Flex>
-                </td>
+
+                  <label htmlFor="isRequired">
+                    <Flex gap={'xs'}>
+                      isRequired
+                      <Checkbox title="Is Required?" {...register('isRequired')} id="isRequired" />
+                    </Flex>
+                  </label>
```

Comment:
If we only use `Checkbox` with title. https://v6.mantine.dev/core/checkbox/#usage